gr/lessons/ecto/associations.md

```diff
@@ -0,0 +1,551 @@
+---
+version: 1.2.1
+title: Συσχετισμοί
+---
+
+Σε αυτό τον τομέα θα μάθουμε πως να χρησιμοποιήσουμε το Ecto για να ορίσουμε και να εργαστούμε με συσχετίσεις ανάμεσα στα σχήματά μας.
```

Comment:
@gemantzu the title is 'συσχετισμοί', but you are using 'συσχετίσεις' here.
Wouldn't it be better if one of them is used?